cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml

```diff
@@ -64,6 +64,17 @@ phases:
               # Install AWS CLI
               which aws
               if [[ $? -ne 0 ]]; then
+                echo "Installing unzip"
+                if [[ ${PLATFORM} == RHEL ]]; then
+                  if [[ ${OS} == centos8 ]]; then
```

Comment:
we should define OS variable in this step.

Reply:
fixed